examples/src/main/java/org/bitcoinj/examples/ForwardingService.java

```diff
@@ -72,7 +76,24 @@ public static void main(String[] args) throws Exception {
         forwardingService.start();
```

Comment:
It would be nice to have a `.start()` that returns a `CompletableFuture<Void>` rather than block, but we can add that later...